geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/StartGatewaySenderCommand.java

```diff
@@ -77,6 +82,9 @@ public ResultModel startGatewaySender(@CliOption(key = CliStrings.START_GATEWAYS
       return ResultModel.createError(CliStrings.NO_MEMBERS_FOUND_MESSAGE);
     }
 
+    // TODO handle error situations when invoking sendAllStartingMessage
+    setNotAllGatewaySenderInstancesStopped(dsMembers, id);
```

Comment:
This method and the latter setAllGatewaySenderInstancesStopped() should be merged into one. And the name is confusing.